ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -91,14 +98,22 @@ def main(argv=sys.argv[1:]):
         start_time = time.time()
 
     compilation_dbs = get_compilation_db_files(args.paths)
+
+    if args.packages_select is not None:
+        # Handle the case of a quoted list of space separated package names
+        if len(args.packages_select) == 1:
+            args.packages_select = args.packages_select[0].strip().split()
+        compilation_dbs = filter_packages_select(compilation_dbs, args.packages_select)
+
     if not compilation_dbs:
         print('No compilation database files found', file=sys.stderr)
         return 1
 
     bin_names = [
         # 'clang-tidy',
-        'clang-tidy-6.0',
+        'clang-tidy-' + args.clang_tidy_version,
```

Comment:
I would make this a separate change. Supporting other versions of `clang-tidy` requires enhancements to the parsing of errors and warnings. The regex at https://github.com/ament/ament_lint/blob/34ef6d2b9cb3c11468d7f5181c4fa17269822188/ament_clang_tidy/ament_clang_tidy/main.py#L184 just won't cut it.

Reply:
Ok, I'll split this change out.  I should have known better.  The reason I did this was that we have standardized on clang-10 tooling for moveit in both ros1 and 2.  As far as I can tell this works with clang-tidy-10.

Reply:
The `stdout` output may be fine, but I think for certain kinds of errors and warning returned by `clang-tidy-10`, the xUnit XML file will contain nonsensical snippets.

Either way, I think adding support for a newer version of `clang-tidy` needs a closer look.

Reply:
I'll split this change out now and do some testing.  Maybe I can figure out a straightforward way to support clang-tidy-10.  It would make using this much easier as 10 is much newer and therefore more easily available in newer Ubuntu now and in the future (hence why we changed in MoveIt).